swift/welcome-email/README.md

```diff
@@ -0,0 +1,68 @@
+# 📧 Sending Welcome Emails using Mailgun's Email API
+
+A sample Swift Cloud Function for sending a welcome email to a newly registered user.
+
+## 📝 Environment Variables
+Go to Settings tab of your Cloud Function. Add the following environment variables.
+
+* **MAILGUN_API_KEY** - API Key for Mailgun 
+* **MAILGUN_DOMAIN** - Domain Name from Mailgun
+
+## 🚀 Building and Packaging
+
+To package this example as a cloud function, follow these steps.
+
+```bash
+$ cd demos-for-functions/swift/welcome-email
+```
+
+* Ensure that your project has the following structure
+```bash
+.
+├── Package.swift
+├── README.md
+└── Sources
+    └── WelcomeEmail
+        ├── File.swift
+        ├── main.swift
+        └── Utils.swift
+
+2 directories, 5 files
+```
+
+* Build the Swift binary using 
+
+```bash
+$ docker run --rm -it -v $(pwd):/app -w /app swift:5.5 swift build
+```
+
+* At this point, if you wish to test the code, you can run it using
+
+```sh
+docker run \
+-e MAILGUN_API_KEY=abcdefg \
+-e MAILGUN_DOMAIN=example.com \
+-e APPWRITE_FUNCTION_EVENT_DATA="{ \"name\": \"John Doe\", \"email\": \"test@test.com\" }" \
+--rm -it -v $(pwd):/app -w /app appwrite/runtime-for-swift:5.5 \
+.build/x86_64-unknown-linux-gnu/debug/WelcomeEmail
```

Comment:
@abnegate I think these packaging steps are not relevant with the new execution model. Can you update it  ?